packages/ui-menu/src/Menu/index.tsx

```diff
@@ -231,87 +199,83 @@ class Menu extends Component<MenuProps> {
     this.setState({ hasFocus: this.focusedIndex >= 0 })
   }
 
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'event' implicitly has an 'any' type.
-  handleMenuItemMouseOver = (event, menuItem) => {
-    // @ts-expect-error ts-migrate(2531) FIXME: Object is possibly 'null'.
-    if (this._activeSubMenu && menuItem !== this._activeSubMenu._trigger) {
+  handleMenuItemMouseOver: MenuItemProps['onMouseOver'] = (event, menuItem) => {
+    if (this._activeSubMenu) {
```

Comment:
The `_trigger` part is missing from the if.

Reply:
This condition will always return 'true' since the types 'MenuItem' and '(ReactElement<any, string | JSXElementConstructor<any>> & { focus?: (() => void) | undefined; }) | null' have no overlap.

If the check is correct (and I think it is), than we don't need that part